.github/workflows/cleanup.yml

```diff
@@ -0,0 +1,22 @@
+name: cleanup
+
+on: 
+  pull_request:
+    types: [closed]
+  
+jobs:
+
+  cleanup:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install AWS CLI
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y awscli
+    - name: Remove files in results sub_dir
+      run: |
+        if aws s3 ls s3://cellpack-results/$GITHUB_REF/ ; then
```

Comment:
this variable is not being defined anywhere, and also probably interpreted as a string, so this won't work as is.